.travis.yml

```diff
@@ -9,7 +9,8 @@ python:
   - "3.7"
 
 install:
-  - make .venv VENV_PYTHON=$TRAVIS_PYTHON_VERSION
+  - make --always-make $VIRTUAL_ENV VENV_PYTHON=python$TRAVIS_PYTHON_VERSION
```

Comment:
remove the old one first perhaps to ensure clean slate?

Reply:
in travis-ci this is fine, each job is ran in a fresh VM or docker image.